### ISSUE TITLE
API Revalidate Example urlPath

### ISSUE THREAD

**Issue by brikiernan** [vercel/platforms#92]
Hello,

I see there is a handler to use on-demand incremental static regeneration which is an awesome feature. I cannot seem to get it working on this platforms example? I've used this feature where the path is straightforward and it works great.

What is the path supposed to look like for Platforms? I see this in the api handler with...

`await res.unstable_revalidate(urlPath);`

I've tried in the body of the post request...

"urlPath": "/_sites/name-of-site" (using the subdomain or customDomian)
"urlPath": "/_sites"
"urlPath": "/"
"urlPath": "/_sites/name-of-site/post-id" (using the slug)

What am I missing here? How should the urlPath look in the request? Or is there some other issue I'm missing in the middleware?

Were you able to test the api/revalidate feature when deployed to Vercel?

Thanks!

**Comment by steven-tey** [comment]
Hey @brikiernan! Feel free to refer to our solution here: https://github.com/vercel/platforms/blob/main/lib/revalidate.ts

What we're doing is basically calling the `/api/revalidate` endpoint of the particular site that you'd like to revalidate the pages for. For instance, if we're revalidating the blog post at `https://demo.vercel.pub/platforms-starter-kit`, we will call:
- `https://demo.vercel.pub/api/revalidate` with `/` as the path
- `https://demo.vercel.pub/api/revalidate` with `/platforms-starter-kit` as the path

In other words, we're handling revalidates on a site-level, not using the `/_sites` path.

Let me know if that makes sense! :)

**Comment by brikiernan** [comment]
Hey @steven-tey -  I figured this out. I missed the fact that the endpoint of the network request needs to match the custom domain or include the subdomain. That's why it was working for me on my local machine and not when deployed to vercel!

This are the 2 lines of code which helped me figure this out for anyone else trying to model this repo and having trouble with the revalidation endpoint.

https://github.com/vercel/platforms/blob/327dfa1bd27ee7ee0ec90181fb96ce4daef714b8/lib/api/post.ts#L236-#L237
